tests/js/harness.js

```diff
@@ -7,10 +7,15 @@ require("gpii-express");
 require("../../");
 
 
-fluid.defaults("gpii.pouch.tests.harness", {
+fluid.defaults("gpii.test.pouch.harness", {
     gradeNames: ["fluid.component"],
-    port:       6789,
-    baseUrl:    "http://localhost:6789/",
+    databases: {
```

Comment:
I've added `loadTestingSupport`.  Calling `gpii.express.loadTestingSupport` from that block as the only means of loading those grades unfortunately results in "assertion outside test case" messages.  We should review together.